content/examples/podcasts.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Sustain Open Source Design Podcast"
+icon: "ti-microphone"
+description: "All about meaningful contributions between Open Source and Design"
+type: "docs"
+---
+
+S.O.S. Design is a podcast dedicated to exploring the intersection of open source and design: how design is crucial in the open source ecosystem, how designers work with coders to make open source software better, and what sustainability means for the field of open source designers. This podcast grew out of the Sustain community and Open Source Design, and seeks to share great conversations with members from both communities and the open source and design space at large.
+
+[Episode 2: Abraham Agiri and Designing as a Coder, and Coding as a Designer](https://sosdesign.sustainoss.org/2)
```

Comment:
Could we make each episode into a header? This will let us work with a table of contents for nicer organization:

```suggestion
## [Episode 2: Abraham Agiri and Designing as a Coder, and Coding as a Designer](https://sosdesign.sustainoss.org/2)
```